packages/libxml2_autotools.rb

```diff
@@ -6,18 +6,18 @@
 class Libxml2_autotools < Autotools
   description 'Libxml2 is the XML C parser and toolkit developed for the Gnome project.'
   homepage 'http://xmlsoft.org/'
-  version Libxml2.version
+  version Libxml2.version.gsub(/-py.*$/, '')
   license 'MIT'
   compatibility 'all'
   source_url 'https://gitlab.gnome.org/GNOME/libxml2.git'
   git_hashtag "v#{version.split('-').first}"
```

Comment:
No need to split here if version is already transformed?

Reply:
It turns out that the libxml2 internal build of the python bindings drop files in the wrong place, so I'm backing out this change.